src/Main.php

```diff
@@ -3,147 +3,157 @@
 namespace Keylogic\Popcatwrapper;
 
 class Main {
-  public function welcome(string $avatar, string $name, string $servername, string $membercount): string {
+  public static function makeServer(string $name, bool $default = true) {
+    if ($default) {
+      $name = str_replace(' ', '+', $name);
+      return "Welcome+To+{$name}!";
+    } else {
+      return str_replace(' ', '+', $name);
+    }
+  }
+  
+  public static function welcome(string $avatar, string $name, string $server, int $membercount, string $background = 'https://cdn.discordapp.com/attachments/850808002545319957/859359637106065408/bg.png'): string {
     $newmembercount = $membercount + 1;
     $servername = str_replace(" ", "+", $servername);
     $name = str_replace(" ", "+", $name);
     $avatar = str_replace(" ", "+", $avatar);
-    return "https://api.popcat.xyz/welcomecard?background=https://cdn.discordapp.com/attachments/850808002545319957/859359637106065408/bg.png&text1=".$name."&text2=Welcome+To+".$servername."&text3=Member+".$newmembercount."&avatar=".$avatar;
+    return "https://api.popcat.xyz/welcomecard?background={$background}&text1={$name}&text2={$server}&text3=Member+{$newmembercount}&avatar={$avatar}";
   }
-  public function color(string $hex): object {
+  
+  public static function color(string $hex): object {
     return json_decode(file_get_contents("https://api.popcat.xyz/color/".$hex));
   }
-  public function lyrics(string $songname): object {
+  public static function lyrics(string $songname): object {
     $songname = str_replace(" ", "+", $songname);
     return json_decode(file_get_contents("https://api.popcat.xyz/lyrics?song=".$songname));

```

Comment:
What if `json_decode()` fails for some reason? This method lacks after-decode validation.

This comment applies to every method that uses `json_decode()`.